sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -203,7 +204,7 @@ namespace Azure { namespace Core {
      */
     template <class T> bool TryGetValue(Key const& key, T& outputValue) const
     {
-      for (auto ptr = m_contextSharedState; ptr; ptr = ptr->Parent)
```

Comment:
Unnecessary change

Reply:
This change was made to silence "Severity	Code	Description	Project	File	Line	Suppression State
Message	lnt-accidental-copy	'auto' doesn't deduce references. A possibly unintended copy is being made"

Is the bar that changes which silence warnings are unnecessary?